airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/upgrade_java_cdk/commands.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import asyncclick as click
+import requests  # type: ignore
+from pipelines.airbyte_ci.connectors.context import ConnectorContext
+from pipelines.airbyte_ci.connectors.pipeline import run_connectors_pipelines
+from pipelines.airbyte_ci.connectors.upgrade_java_cdk.pipeline import run_connector_java_cdk_upgrade_pipeline
+from pipelines.cli.dagger_pipeline_command import DaggerPipelineCommand
+
+
+@click.command(cls=DaggerPipelineCommand, short_help="Upgrade Java CDK version")
+@click.argument("target-java-cdk-version", type=str, default="")
+@click.pass_context
+async def upgrade_java_cdk_version(
```

Comment:
Can we avoid declaring a language specific command to upgrade the CDK version? The options are the same as the existing upgrade_cdk command.
I'd prefer forking the logic according to connector language [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/upgrade_cdk/pipeline.py#L86)